LogIn.php

```diff
@@ -52,6 +52,7 @@
                       $rowPass = $row['password'];
                       if($rowPass == $AdminPassword){
                              $_SESSION['entered']='entered';
+                             $_SESSION['name']="".$row['first_name']." ".$row['last_name']."";
```

Comment:
added session to store name, so in admin home page we don't need to retrieve it from the database again

Reply:
yes!
good for less code implementation and saves time